azuredisk-csi.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: azuredisk-csi
```

Comment:
seems upstream supports multiple versions this is a candidate for version stream

Reply:
thanks @ajayk , I have modified the file name as azuredisk-ci-1.31 and as discussed with @EyeCantCU and others in team on our working session, I will add a manual version stream for now till I create PRs in end of life and add the remaining supported packages in enterprise